src/main/java/de/dhbw/mh/slang/codegen/JavaBytecodeGenerator.java

```diff
@@ -17,7 +16,7 @@
 
 public class JavaBytecodeGenerator implements AstVisitor<String> {
 	
-	private int counterIDLOR = 0;
+	private int labelCounter = 0;
```

Comment:
Thank you for being so thoughtful to use the existing counter and rename it appropriately 👍